src/ztc/datamodel/models/zaken.py

```diff
@@ -84,10 +88,10 @@ def clean(self):
         """
         super().clean()
 
-        if self.ander_objecttype == JaNee.nee and self.objecttype not in ObjectTypen.values.keys():
+        if not self.ander_objecttype and self.objecttype not in ZaakobjectTypes.values.keys():
```

Comment:
Vaag hoe deze er door is geglipt zonder fouten...

Reply:
test is wss disabled ervoor, als die er al is... :)